TYPO3.Neos/Resources/Public/JavaScript/Content/Inspector/Section.js

```diff
@@ -44,7 +47,7 @@ define(
 			if (!this.get('inspector.configuration.' + this.get('_nodeType'))) {
 				this.set('inspector.configuration.' + this.get('_nodeType'), {});
 			}
-			this.set('inspector.configuration.' + this.get('_nodeType') + '.' + this.get('group'), this.get('_collapsed'));
+			this.set('inspector.configuration.' + this.get('_nodeType') + '.' + this.get('groupName'), this.get('_collapsed'));
```

Comment:
@radmiraal thanks for the review! Previously `group` contained the group name because in the `InspectorView.html` there was some `groupNameBinding="group.group"`.

I thought that a) this was confusing and that `groupName` was more descriptive and b) I needed access to the actual `group` object which wasn't available in `Section`. Alternatively I could have created a `collapsedBinding`, but I didn't really like it.

But of course I may have missed something completely – you tell me! ;-)